tools/blue_deployment/step1.js

```diff
@@ -0,0 +1,343 @@
+ *
+ * @param {Address} adminAddress Address of the organization admin.
+ * @param {Address} ownerAddress Address of the organization owner.
+ * @param {Array.<Address>} workerAddresses Addresses of the organization workers.
+ * @param {expirationHeight} expirationHeight Block height at which the
+ *                           workers will expire.
+ *
+ * @returns {Contract} The Organization contract instance.
+ */
+const createOrganization = (adminAddress, ownerAddress, workerAddresses, expirationHeight) =>  {
+    return Contract.loadTruffleContract('Organization', [
+        adminAddress,
+        ownerAddress,
+        workerAddresses,
+        expirationHeight,
+    ], { rootDir });
+};
+
+/**
+ * Create an Anchor Contract
+ *
+ * @param {Address|ContractReference} organization The address of or reference
+ *                                    to the Organization contract for the anchor.
+ * @param {string} remoteChainId The chain id of the remote chain to anchor.
+ * @param {number} remoteBlockHeight The block height of the remote chain to anchor.
+ * @param {string} remoteStateRoot The state root of the remote chain to anchor.
+ * @param {number} maxStateroots The max number of state roots to store in
+ *                               thecircular buffer.
+ *
+ * @returns {Contract} The Anchor contract instance.
+ */
+const createAnchor = (
+    organization,
+    remoteChainId,
+    remoteBlockHeight,
+    remoteStateRoot,
+    maxStateroots = '10',
+) => {
+    return Contract.loadTruffleContract(
+        'Anchor',
+        [
+            remoteChainId,
+            remoteBlockHeight,
+            remoteStateRoot,
+            maxStateroots,
+            organization,
+        ],
+        { rootDir },
+    );
+};
+
+const createEIP20Gateway = (
+    tokenAddress,
+    baseTokenAddress,
+    anchorAddress,
+    bounty,
+    organizationAddress,
+    burnerAddress = '0x0000000000000000000000000000000000000000',
+) => {
+    return Contract.loadTruffleContract(
+        'EIP20Gateway',
+        [
+            tokenAddress,
+            baseTokenAddress,
+            anchorAddress,
+            bounty,
+            organizationAddress,
+            burnerAddress,
+        ],
+        { rootDir },
+    );
+};
+
+// tokenAddress = Branded Token
+// baseTokenAddress = OST
+const deployAnchorAndGateway = async (
+    rpcEndpointOrigin,
+    deployerAddress,
+    tokenAddress,
+    baseTokenAddress,
+    bounty,
+    chainIdAuxiliary,
+    blockHeightAuxiliary,
+    stateRootAuxiliary,
+) => {
+    const web3 = new Web3(rpcEndpointOrigin);
+    const startingNonce = await web3.eth.getTransactionCount(deployerAddress);
+
+    const {
+        MerklePatriciaProof,
+        GatewayLib,
+        MessageBus,
+    } = createLibraryConracts();
+    const Organization = createOrganization(
+        deployerAddress, // TODO
+        deployerAddress, // TODO
+        [deployerAddress], // TODO
+        '100000000000', // TODO
```

Comment:
Please remove `TODO` comments. Either fix or create follow-up ticket(s) (generally).

Reply:
How about TODO with number/link to the follow up tickets? Makes it easier to both find the issue for a part of the code that needs work, and also makes it easier to find the code again (if it's being moved around) based on the issue.

Reply:
The problem with that is that I search for `TODO` to find anything that I still have to do before creating the PR. And this way I end up with a lot of false-positives. But if I am the only one doing this, then it is ok. I can use something else.
Another suggestion would be using `FIXME` with the ticket number. What do you think?

Reply:
Good point! Yeah I normally also do that (though I usually do `git diff | grep TODO' which yields less false-positives but also misses stuff in an earlier commit of a PR, as it did here).

`FIXME` + ticket number sounds good 👍

Reply:
👍

Reply:
For this case #623 has been created as a ticket